app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
Thanks @hweng 👍

To answer your question: yep, that has happened in the past. @lsitu has dealt with that a few times. This is often with collections that were ingested many years ago, I believe.

_Hopefully_ re-running JHOVE will generate updated correct mimetype for most of the images, though re-creating some derivatives may be necessary.